src/couch_mrview/src/couch_mrview.erl

```diff
@@ -315,14 +315,16 @@ count_view_changes_since(Db, DDoc, VName, SinceSeq, Options) ->
                 _ -> View#mrview.seq_btree
             end,
             lists:foldl(fun(Opts, Acc0) ->
-                            {ok, N} = couch_btree:fold_reduce(
-                                    Btree, fun(_SeqStart, PartialReds, 0) ->
-                                        {ok, couch_btree:final_reduce(
-                                                    Btree, PartialReds)}
-                                    end,
-                                0, Opts),
-                            Acc0 + N
-                    end, 0, OptList);
+                case couch_btree:fold_reduce(Btree,
+                    fun(_SeqStart, PartialReds, 0) ->
```

Comment:
Big narp on this formatting. Assign your anonymous function to a variable and pass that.